modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1731,6 +1737,34 @@ public static double getDouble(String name, double dflt) {
         return res;
     }
 
+    /**
+     * Gets either system property or environment variable with given name.
+     * The result is transformed to {@link UUID} using {@link UUID#fromString} method.
+     *
+     * @param name Name of the system property or environment variable.
+     * @param dflt Default value.
+     * @return UUID value of the system property or environment variable.
+     *         Returns default value in case neither system property
+     *         nor environment variable with given name is found.
+     */
+    public static UUID getUUID(String name, UUID dflt) {
+        String s = getString(name);
+
+        if (s == null)
+            return dflt;
+
+        UUID res;
+
+        try {
+            res = UUID.fromString(s);
+        }
+        catch (IllegalArgumentException ignore) {
+            res = dflt;
```

Comment:
I'd avoid swalloing the exception here. At least a warning would be nice

Reply:
It is default behavioral for the whole class (`getInteger`, `getLong`, etc.). I don't think we should change it specifically for this method.